CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+- Be a liaison for your project with our group
+  - Why? The ecosystem is very large, if you stay connected, you’ll be closer to
+resources and advice.
+  - Take back this information to build (or continue to build!) intentional
+community and governance spaces and/or roles in your project to do this work at
+an operational level
+- Join us for a Contributor Strategy AMA on the 4th Thursday meeting of each month
+ at the last :30 minutes as you need. This is your time to ask us general or
+ specific questions that can give you ideas or unblock you. The other meeting
+ time we have is devoted to meta SIG and working group planning.
+- Give us feedback on where you need help, guidance, requested templates,
+recommendations, and more here
+Or you can provide feedback by filing issues against the
+cncf/sig-contributor-strategy repo, send a note to the mailing list, or find a
+chair or working group facilitator privately on slack.
+- Become a contributor in one of our [working groups]
+  - File issues and PRs
+  - Project management
+- Create workshops, talks, blogs, and papers on related contributor content (or
+  reuse the good stuff you have!)
+- Create templates and other scalable guidance
+- Tell us what WORKS! Sounds like a lot of “what doesn’t work” above but we also
+want to know what works well for projects in their contributor and governance
+areas. Looking at you graduated projects but we see the incubation project crew
+still working on great programs!
+- Be a part of our work in progress maintainers circle! [link to issue]
+
+Academics and open source enthusiasts that have completed research (or underway!)
+, studied, and/or created programs for other ecosystems, we would love to hear
+from you, too! Similar deal as above - watch out for our Issues, come hang out
+in slack, or we'll see you at a meeting!
+
+### Tips for filing issues
+//TODO add in comments about how to label once we figure that out
+
+### Pull Request Tips - Submitting and Reviewing
+- Talk to us via GitHub issue, mailing list, or meeting before you start on major
+work. If it’s typos and fixes to docs, please submit. 😀
+- Tap one of the chairs or working group facilitators in slack if you haven’t
+heard from us in 5 days.
+- When making changes to your PR, put them into a new commit (instead of rebasing
+  or amending) so that it's easier for reviewers to see just your new changes.
+- We will give you an opportunity to clean up your commits before merging or can
+squash them for you.
+- If a member asks you to "rebase" your PR, they're saying that a lot of docs/code
+ has changed, and that you need to update your branch to incorporate the latest
+ changes from master before we can merge you PR. You can use whatever you are
+ comfortable with, either merging master into your branch or rebasing your
+ branch on master.
+#### Reviewers:
+- We leave pull requests open for at least 48 hours, so that others have the
+chance to review/comment except in cases of the following:
```

Comment:
Is this where we should have approval workflow?  Or is that a different doc?

Reply:
here

Reply:
per discussion on call today, we're gonna spin off the approval workflow into a separate doc